File: HW1 (Simple Programs)/Scala/HW1_4.scala
object HW1_4 {
	def main(args : Array[String]) {
		print("Input: ")
		var x = scala.io.StdIn.readDouble()
		println(f(x))
	}

	def f(x : Double) : Double = {
		return x * x + (math.log10(x) - math.sin(x)) / math.sqrt(x)
	}
}

File: HW1 (Simple Programs)/C#/HW1_4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSC617M {
	class HW1_4 {
		static void Main(string[] args) {
			Console.Write("Enter a value: ");
			double x = double.Parse(Console.ReadLine());
			double ans = x*x + (Math.Log(x) - Math.Sin(x))/Math.Sqrt(x);
			Console.WriteLine("Answer is " + ans + "!\n");
		}
	}
}